README.md

```diff
@@ -1 +1,19 @@
-# go-launch-a-survey
\ No newline at end of file
+# Pre-Alpha: Go Launch a Survey!
+
+### Building and Running
+Install Go and ensure that your `GOPATH` env variable is set (usually it's `~/go`).
+
+```
+go get -d github.com/collisdigital/go-launch-a-survey/
+cd $GOPATH/src/github.com/collisdigital/go-launch-a-survey/
+go build
```

Comment:
Well if you will 'go' against the grain; apparently that's where all the golaners do it